README.md

```diff
@@ -15,6 +15,102 @@ Our aim is to infuse these strengths into the Ethereum consensus client ecosyste
 
 We also have for objective to bootstart an Ethereum Elixir community, and to make Elixir a first-class citizen in the Ethereum ecosystem.
 
+## Contributor Package 
+
+Dream of becoming an Ethereum core developer? Eager to shape the protocol that will underpin tomorrow's world? Want to collaborate with a passionate team, learn, grow, and be a pivotal part of the Ethereum Elixir community? 
+
+**Then you're in the right place! 🚀**
+
+### Getting Started:
+
+#### 1. **Installation**:
+
+- **Prerequisites**: Before diving in, ensure you have the necessary tools installed. Check out the [Prerequisites](#prerequisites) section for guidance.
+  
+- **Clone the Repository**:
+  ```shell
+  git clone [REPO_URL]
+  cd lambda-ethereum-consensus
+  ```
+
+- **Setup**: Once you've cloned the repository, follow the steps in the [Installing and running](#installing-and-running) section to set up your environment.
+
+#### 2. **Prerequisite Knowledge**:
+
+To contribute effectively, you'll need a foundational understanding of both the Ethereum protocol and the Elixir language, including the Erlang VM (BEAM). If you're new to these areas, we've curated a list of resources to get you started:
+
+**Learning Elixir**:
+- **Books**:
+  - [Elixir in Action](https://www.manning.com/books/elixir-in-action-third-edition)
+  - [Learn You Some Erlang](https://learnyousomeerlang.com/)
+- **Videos**:
+  - [Intro to Elixir](https://youtube.com/playlist?list=PLJbE2Yu2zumA-p21bEQB6nsYABAO-HtF2)
+  - [Hitchhiker's tour of the BEAM](https://www.youtube.com/watch?v=_Pwlvy3zz9M)
+- **Blogs**:
+  - [Zen of Erlang](https://ferd.ca/the-zen-of-erlang.html)
+  - [Where Erlang Blooms](https://ferd.ca/rtb-where-erlang-blooms.html)
+  - [What can I only do in Erlang](https://hackmd.io/ZpUazPomRvacKoyW2vq54g)
+  - [Stacking theory for systems design](https://medium.com/@jlouis666/stacking-theory-for-systems-design-2450e6300689)
+  - [On Erlang States and Crashes](http://jlouisramblings.blogspot.com/2010/11/on-erlang-state-and-crashes.html)
+  - [How Erlang does scheduling](http://jlouisramblings.blogspot.com/2013/01/how-erlang-does-scheduling.html)
+
+With this foundation you should have a basic understanding of the Elixir language and the Erlang VM. You can then start (or in parallel) learning about the Ethereum protocol.
+
+**Learning Ethereum**:
+- **Books**:
+  - [Mastering Ethereum](https://github.com/ethereumbook/ethereumbook)
```

Comment:
I would not add this one, it seems quite unrelated to the consensus part that we're working here

Reply:
done